frontend/src/components/Options.vue

```diff
@@ -16,6 +16,8 @@
 
         <b-dropdown-item @click="claimSkill(ClaimStage.WaxBridge)">Claim Skill </b-dropdown-item>
 
+        <b-dropdown-item @click.native="$router.push('leaderboard')">Leaderboard </b-dropdown-item>
```

Comment:
this will need the google analytics stuff - `gtag-link-others` and `tagname` (I think that's all it needs to be picked up).

Reply:
added